modules/color-picker/react/lib/ColorInput.tsx

```diff
@@ -133,11 +133,12 @@ export default class ColorInput extends React.Component<ColorInputProps> {
           inputRef={inputRef}
           onChange={this.handleChange}
           type="text"
-          placeholder={placeholder}
+          placeholder={value ? '' : placeholder}
```

Comment:
For some reason the placeholder text was being announced when the input had a value. If there is a value, i set the placeholder text to empty, which just announces the editable text. It only announces `FFFFFF` when there is no value in the input.

Reply:
Interesting. Many times accessibility guidelines recommend removing attributes vs setting them to `false` or empty values. Did you try that?

```suggestion
          placeholder={value ? undefined : placeholder}
```

Reply:
ooo haven't tried that, i can give it a go